src/api/zodClient.ts

```diff
@@ -778,6 +778,15 @@ const FeatureOverride = z.object({
 const FeatureOverrideResponse = z.object({
     overrides: z.array(FeatureOverride)
 })
+const UserOverride = z.object({
```

Comment:
do you not need to add this as a schema object? of is that only if we need to use the type elsewhere?

Reply:
added to the object in case we need it in the future, but not needed since this is response type and not a payload type / something that we'd use to validate sending payloads to the API

Reply:
that makes sense!